Quick_Deploy/HuggingFaceTransformers/README.md

```diff
@@ -0,0 +1,180 @@
+
+```
+I0922 23:28:40.351809 1 grpc_server.cc:2451] Started GRPCInferenceService at 0.0.0.0:8001
+I0922 23:28:40.352017 1 http_server.cc:3558] Started HTTPService at 0.0.0.0:8000
+I0922 23:28:40.395611 1 http_server.cc:187] Started Metrics Service at 0.0.0.0:8002
+```
+
+## Step 4: Use a Triton Client to Query the Server
+
+Now we can query the server using curl, specifying the server address and input details:
+
+```json
+curl -X POST localhost:8000/v2/models/falcon7b/infer -d '{"inputs": [{"name":"prompt","datatype":"BYTES","shape":[1],"data":["I am going"]}]}'
+```
+In our testing, the server returned the following result (formatted for legibility):
+```json
+{
+  "model_name": "falcon7b",
+  "model_version": "1",
+  "outputs": [
+    {
+      "name": "text",
+      "datatype": "BYTES",
+      "shape": [
+        1
+      ],
+      "data": [
+        "I am going to be in the market for a new laptop soon. I"
+      ]
+    }
+  ]
+}
+```
+
+## Step 5: Host Multiple Models in Triton
+
+So far in this tutorial, we have only loaded a single model. However, Triton is capable
+of hosting many models, simultaneously. To accomplish this, first ensure you have
+exited the docker container by invoking `Ctrl+C` and waiting for the container to exit.
+
+Next copy the remaining models provided into the model repository:
+```
+cp -r mistral7b/ model_repository/
+cp -r persimmon8b/ model_repository/
+```
+*NOTE*: The combined size of these three models is large. If your current hardware cannot
+support hosting all three models simultaneously, consider copying only a single additional
+model.
+
+Again, launch the server by invoking the `docker run` command from above and wait for confirmation
+that the server has launched successfully.
+
+Query the server making sure to change the host address for each model:
+```json
+curl -X POST localhost:8000/v2/models/falcon7b/infer -d '{"inputs": [{"name":"prompt","datatype":"BYTES","shape":[1],"data":["How can you be"]}]}'
+curl -X POST localhost:8000/v2/models/mistral7b/infer -d '{"inputs": [{"name":"prompt","datatype":"BYTES","shape":[1],"data":["Where are you going"]}]}'
+curl -X POST localhost:8000/v2/models/persimmon8b/infer -d '{"inputs": [{"name":"prompt","datatype":"BYTES","shape":[1],"data":["Where is the nearest"]}]}'
+```
+In our testing, these queries return the following parsed results:
+```bash
+# falcon7b
+"How can you be sure that you are getting the best deal on your car"
+
+# mistral7b
+"Where are you going? I’m going to the beach."
+
+# persimmon8b
+"Where is the nearest starbucks?"
+```
+## 'Hour Zero' Support
+
+At the time of writing this tutorial, transformers version 4.34.0 was not yet released, meaning
+very new models such as Persimmon-8B and Mistral 7B were not yet fully supported by the latest
+transformers releases (4.33.3). Triton is not limited to waiting for official releases and can
+load the latest models by building from source. This can be done by replacing:
```

Comment:
We may want to avoid specific versions and models here to avoid dating it too much, as these models mentioned will quickly become "old" compared to newer ones. We should instead just say something more generic about how bleeding edge / latest models may not be supported in the latest published release of `transformers`, so for bleeding edge model support you can install latest transformers from source. To do so, replace the transformers install in the `Dockerfile` with `pip install git+https://github.com/huggingface/transformers.git` ...

Using this technique you should be able to serve any models supported by HuggingFace Transformer text generation pipeline with Triton.

---

above is just the gist, feel free to rephrase, just want to avoid mention of specific versions and models in this disclaimer

Reply:
Great point!